tools/src/main/java/org/apache/kafka/trogdor/workload/Payload.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.trogdor.workload;
+
+import java.util.Random;
+
+/**
+ * Describes the payload for the producer record. Currently, we can configure message size
+ * (constant), expected compression rate. Where value is of message size, and key is null.
+ * We will likely make this class a base class in the future and derive different payload classes
+ * for various message size distributions, key assignments, etc.
+ */
+public class Payload {
```

Comment:
Could we have other types of Payload? Should this be `ProducerPayload`?

Reply:
Yes, I will rename it to ProducerPayload -- this controls how we populate keys and values. We don't need to do anything similar for consumers.